nestkernel/connector_base.h

```diff
@@ -177,6 +212,24 @@ class Connector : public vector_like< ConnectionT > // unfortunately, we need th
     C_[ K - 1 ] = c;
   }
 
+  /*
+   * Create a new connector and remove the ith connection. Returns a connector
+   * with size K from a connector of size K+1.
+   */
```

Comment:
Please add some more explanation about the copying process implemented below. It took me quite a bit of time to figure it out. Always expect your readers to be as tired as I am currently ;-)